lte/gateway/c/core/oai/lib/pipelined_client/PipelinedClientAPI.cpp

```diff
@@ -42,8 +42,14 @@ int upf_classifier_add_tunnel(
     struct in_addr ue, struct in6_addr* ue_ipv6, int vlan, struct in_addr enb,
     uint32_t i_tei, uint32_t o_tei, const char* imsi,
     struct ip_flow_dl* flow_dl, uint32_t flow_precedence_dl, const char* apn) {
-  std::string imsi_str = std::string(imsi);
-  std::string apn_str  = std::string(apn);
+  std::string imsi_str = "";
```

Comment:
when is imsi null?

Reply:
Not sure when IMSI came as NULL, but for secure better put the null check.

Reply:
It comes from MME correct? @ardzoht any idea when imsi can be null in this case? I think we should track this and investigate if this is not expected behaviour